compiler/arm/codegen/ControlFlowEvaluator.cpp

```diff
@@ -1812,6 +1812,7 @@ static TR::Register *generateMaxMin(TR::Node *node, TR::CodeGenerator *cg, bool
 
    TR_ARMConditionCode cond = max ? (isUnsigned ? ARMConditionCodeHI : ARMConditionCodeGT) : (isUnsigned ? ARMConditionCodeLS : ARMConditionCodeLT);
    int n = node->getNumChildren();
+   TR_ASSERT(n == 2, "Expected 2 children only");
```

Comment:
Is this change related to the PR? Similarly for the other `TR_ASSERT` changes?

Reply:
I think Aidan has accidentally pushed a few extra commits, these compiler changes weren't present initially.

Reply:
Apologies, I did a rebase off of my master, and that branch had some extra commits